File: DemoApp/module1/src/main/java/com/example/module1/MainActivity.java
package com.example.module1;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    EditText Username;
    EditText Pass;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Username = findViewById(R.id.editText);
        Pass = findViewById(R.id.editText2);

    }


        public void onButtonClicked2(View view) {

            String user = Username.getText().toString();
            String pass = Pass.getText().toString();
            if (user.isEmpty()){
                Toast.makeText(this, "User name not found", Toast.LENGTH_LONG).show();
            }
            else if (pass.isEmpty()){
                Toast.makeText(this, "Password Not found", Toast.LENGTH_LONG).show();
            }
            else {
                Intent intent = new Intent(MainActivity.this, main_menu.class);
                intent.putExtra("Username",user);
                startActivity(intent);

            }

    }

    public void onButtonClicked3(View view) {


        Toast.makeText(MainActivity.this, "Go to Signup page", Toast.LENGTH_LONG).show();

    }

}

File: KotlinApplication/app/src/main/java/com/example/kotlinapplication/MainActivity.kt
package com.example.kotlinapplication

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.AppCompatTextView

class MainActivity : AppCompatActivity() {

    lateinit var mUserName: AppCompatTextView
    lateinit var mEmail: AppCompatTextView
    lateinit var mPhone: AppCompatTextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        mUserName = findViewById(R.id.mainUser) as AppCompatTextView
        mEmail = findViewById(R.id.mainEmail) as AppCompatTextView
        mPhone = findViewById(R.id.mainPhone) as AppCompatTextView

        val intent = intent

        var userName: String = intent.getStringExtra("user")
        var email: String = intent.getStringExtra("email")
        var phone: String = intent.getStringExtra("phone")

        mUserName.text = userName
        mEmail.text = email
        mPhone.text = phone

    }
}
